(#12) Create a preliminary Aerodynamic Model

Nufflee opened the pull request:
This PR extracts aerodynamic calculations from Derivatives.m file and makes Cd and Cl placeholder functions based on the [Thin Airfoil Theory](https://en.wikipedia.org/wiki/Airfoil#Thin_airfoil_theory) parametrized by the AoA (Alpha).
Also fixes a few formatting and clarity issues.

Closes #12.

Nufflee committed:
Extract aerodynamic calculations to AerodynamicModel.m

Nufflee committed:
Make Cd and Cl functions of AoA

This doesn't seem to work yet because there are some AoA anomalies.

Nufflee committed:
Fix a typo and reformat Get_FSM_State

* Fix `<-` and `>-` typo
* Fix bug wher Get_FSM_State could return two states at the same time if velocity is exactly 5 m/s

Nufflee committed:
Change casing of Get_FSM_State file to be consistent with the function

ahh, sometimes I hate Windows and its case insensitive filenames..

Nufflee committed:
Use `disp` instead of `printf` for clarity

Nufflee committed:
Fix disp syntax error

TheGreatFez commented on Simulation/Get_FSM_State.m:
```diff
@@ -7,15 +7,15 @@
-    crashed =(height<=ground_height) && (v_y<-5);
-    on_ground = (height<=ground_height) && (v_y>-5);
+    flying = (height > ground_height) && (v_x > 0);
+    crashed = (height <= ground_height) && (v_y <= 5);
```
This needs to be -5, +5 means you are flying upward

TheGreatFez commented on Simulation/Get_FSM_State.m:
```diff
@@ -7,15 +7,15 @@
-    on_ground = (height<=ground_height) && (v_y>-5);
+    flying = (height > ground_height) && (v_x > 0);
+    crashed = (height <= ground_height) && (v_y <= 5);
+    on_ground = (height <= ground_height) && (v_y > 5);
```
Same as above

TheGreatFez commented on Simulation/AerodynamicModel.m:
```diff
@@ -0,0 +1,20 @@
+  Velocity = StateVector(3:4);
+  Speed = norm(Velocity); % Magnitude of velocity vector
+  
+  Density = 1.225;                              % Standard air density (kg/m3)
```
This density should be grabbed from the Atmosphere Model

TheGreatFez reviewed (changes requested):
Couple of changes needed on some minor things, other than that good job!

Nufflee committed:
Fix FSM state bug

Fix typo I introduced earlier where the landing speed was missing a negative sign.

Nufflee committed:
Use the atmosphere model for air density in the aerodynamic model

We now use the correct values form the atmospheric model in the aerodynamic model based on the current altitude of the plane from the StateVector.

TheGreatFez reviewed (approved):
Approved, nice work!
